### PR TITLE
Enable FrozenStringLiteralComment

### PR DESCRIPTION
**Pull request by anduonghien**
## What happened
Solved #119

 
## Insight
1/ The FrozenStringLiteralComment is enabled by default https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/FrozenStringLiteralComment

2/ Also fixing some typo, we can use a single quote instead of double quote 
 
## Proof Of Work

The codebase specs are passed

image

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -2,15 +2,15 @@ require: rubocop-rails
...
-    - 'vendor/**/*'
-    - 'db/**/*'
-    - 'bin/**/*'
-    - 'Gemfile'
-    - 'Gemfile.lock'
-    - 'Rakefile'
-    - 'Guardfile'
-    - 'node_modules/**/*'
-    - 'config/**/*'
+    - "vendor/**/*"
```
Comment (hoangmirs) [inline .rubocop.yml:5]: Why do we change to use double quote?

**Review by hoangmirs — COMMENTED** [review]
You can change the PR description from `Solved #119` to `Resolve #119`.
It will [close that issue](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) automatically for you once you merge this PR.

File: .rubocop.yml (hunk 1 above)
Comment (anduonghien) [reply .rubocop.yml]: Oh it done by my VSCode. will rever it @hoangmirs

Comment (anduonghien) [reply .rubocop.yml]: Reverted in https://github.com/nimblehq/rails-templates/pull/198/commits/ce7fa211ada4aeeb2881ffea2d61c64d6c9342fc

**Review by hoangmirs — APPROVED** [review]

**Review by longnd — APPROVED** [review]

**Pull request merged**